src/sentry/static/sentry/app/views/organizationEventsV2/data.tsx

```diff
@@ -12,14 +12,17 @@ import getDynamicText from 'app/utils/getDynamicText';
 import overflowEllipsis from 'app/styles/overflowEllipsis';
 import pinIcon from 'app/../images/location-pin.png';
 import space from 'app/styles/space';
+import {EventView} from 'app/types';
 
 import {QueryLink} from './styles';
 
-export const MODAL_QUERY_KEYS = ['eventSlug'];
+export const MODAL_QUERY_KEYS: ['eventSlug'] = ['eventSlug'];
```

Comment:
No better way not to duplicate this?

Reply:
I'll adjust this. 

This works well with `as const`.